src/components/organisms/ActiveTradeTable/ActiveTradeTable.test.tsx

```diff
@@ -93,8 +93,9 @@ describe('ActiveTradeTable Component', () => {
     });
 
     it('should display hours and minutes when maturity is less than 24 hours', async () => {
-        render(<Default />);
+        render(<Delisted />);
```

Comment:
@as472 
The test kept failing and when looking at storybook, I see this belongs to the Delisted story instead of Default.
```
expect(closeToMaturityRow).toHaveTextContent('Feb 2, 2022');
waitFor(() => {
     expect(closeToMaturityRow).toHaveTextContent('21h-59m');
});
```